components/browser/engine-gecko-beta/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineTest.kt

```diff
@@ -210,6 +210,51 @@ class GeckoEngineTest {
         verify(mockRuntime.settings.contentBlocking).setStrictSocialTrackingProtection(true)
     }
 
+        @Test
+    fun `WHEN a strict tracking protection policy is set THEN the setEnhancedTrackingProtectionLevel must be STRICT`() {
+        val mockRuntime = mock<GeckoRuntime>()
+        whenever(mockRuntime.settings).thenReturn(mock())
+        whenever(mockRuntime.settings.contentBlocking).thenReturn(mock())
+
+        val engine = GeckoEngine(testContext, runtime = mockRuntime)
+
+        engine.settings.trackingProtectionPolicy = TrackingProtectionPolicy.strict()
+
+        verify(mockRuntime.settings.contentBlocking).setEnhancedTrackingProtectionLevel(
+            ContentBlocking.EtpLevel.STRICT
+        )
+    }
+
+    @Test
+    fun `WHEN a recommended tracking protection policy is set  THEN the setEnhancedTrackingProtectionLevel must be DEFAULT`() {
+        val mockRuntime = mock<GeckoRuntime>()
+        whenever(mockRuntime.settings).thenReturn(mock())
+        whenever(mockRuntime.settings.contentBlocking).thenReturn(mock())
+
+        val engine = GeckoEngine(testContext, runtime = mockRuntime)
+
+        engine.settings.trackingProtectionPolicy = TrackingProtectionPolicy.recommended()
+
+        verify(mockRuntime.settings.contentBlocking).setEnhancedTrackingProtectionLevel(
+            ContentBlocking.EtpLevel.DEFAULT
+        )
+    }
+
+    @Test
+    fun `WHEN a tracking protection policy different recommended or strict from is set to recommended THEN the setEnhancedTrackingProtectionLevel must be NONE`() {
```

Comment:
nit: I think this test name needs to be rephrased? I would have just combined this all in one test with comments as you're just testing the different combinations...if you don't want that then how about something like 'WHEN a tracking protection policy other than recommended or strict from is set THEN the setEnhancedTrackingProtectionLevel must be NONE'.